src/plugins/console/public/application/containers/main/get_tour_steps.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { EuiStatelessTourSteps } from '@elastic/eui';
+
+const commonProps = {
+  stepsTotal: 5,
+  maxWidth: 400,
+};
+
+export function getTourSteps() {
+  return [
+    {
+      step: 1,
+      title: i18n.translate('console.tour.shellStepTitle', {
+        defaultMessage: 'Welcome to the Console',
+      }),
+      content: i18n.translate('console.tour.shellStepContent', {
+        defaultMessage:
+          'This is our UI for interacting with Elasticsearch clusters using QueryDSL. Easily run queries, manage settings, and view search results.',
+      }),
+      anchorPosition: 'upLeft',
+      'data-test-subj': 'shellTourStep',
+      ...commonProps,
+    },
+    {
+      step: 2,
+      title: i18n.translate('console.tour.editorStepTitle', {
+        defaultMessage: 'Get started querying',
+      }),
+      content: i18n.translate('console.tour.editorStepContent', {
+        defaultMessage:
+          'Enter a request in this input pane, and the response will be shown in the neighboring output pane. For more details, visit our QueryDSL documentation.',
+      }),
+      anchorPosition: 'leftUp',
+      'data-test-subj': 'editorTourStep',
+      css: {
+        position: 'absolute',
+        top: '40%',
+        left: '15%',
+        transform: 'translateY(10px)',
+      },
+      ...commonProps,
+    },
+    {
+      step: 3,
+      title: i18n.translate('console.tour.historyStepTitle', {
+        defaultMessage: 'Revisit past queries',
+      }),
+      content: i18n.translate('console.tour.historyStepContent', {
+        defaultMessage:
+          'Enter a request in this input pane, and the response will be shown in the neighboring output pane. For more details, visit our QueryDSL documentation.',
```

Comment:
I think this copy should be: `The history panel keeps track of your past queries, making it easy to review and rerun them.`

Reply:
Nice catch! Fixed in [f64b854](https://github.com/elastic/kibana/pull/191021/commits/f64b854c35c615a5039b9cc826b74ce0e257c19a).